requirements/development.txt

```diff
@@ -35,3 +35,7 @@ django_coverage_plugin==3.1.0
 
 # Used to build the ML model
 requests-cache==0.9.5
+
+# CORS headers
```

Comment:
These are required in production right? Probably want both of these in `base.txt` right?